lib/aruba/environment/unix_environment.rb

```diff
@@ -0,0 +1,6 @@
+require 'aruba/environment/basic_environment'
+
+module Aruba
+  class UnixEnvironment < BasicEnvironment
```

Comment:
I don't think it's a good idea to use *Posix* instead of Unix:
- POSIX is a set of compliance standards that applies to many operating systems.  It's not == Unixe or Linux. If an OS meets the Posix standard, it's "Posix compliant."  Many flavors of *nix systems are Posix compliant. I know that Posix is often used as a shorthand for a group of *nix systems, but that's using the term pretty loosely. (And incorrectly)  It's important here to be more accurate.
- We don't have a way to test to see if a system *"is posix?"*  (Which really means "is POSIX compliant.")  FFI doesn't have a test for it.(We _could_ implement a valid test. we'd have to get the list of Posix-compliant systems and ensure that info is kept updated.) 
 Someone reading through the code would see <code>posix?</code> and assume that we're using a valid test for Posix. -- but we're not.    Even if we just use the test <code> !windows? </code> we still need to name the class.

So I would argue that using Posix isn't really the way to go here. 
I hope I'm not sounding argumentative -- I don't mean to.  Just want to make sure everything is considered.